src/server/enterprise/server/api_server.go

```diff
@@ -330,3 +343,192 @@ func (a *apiServer) Deactivate(ctx context.Context, req *ec.DeactivateRequest) (
 
 	return &ec.DeactivateResponse{}, nil
 }
+
+func (a *apiServer) Pause(ctx context.Context, req *ec.PauseRequest) (resp *ec.PauseResponse, retErr error) {
+	if err := scaleDownWorkers(ctx, a.env.GetKubeClient(), a.env.Namespace); err != nil {
```

Comment:
Do we still need this call here since it's scaling down workers on startup of a cluster in `paused` mode?

Reply:
I thought of it as belt-and-suspenders, on the theory that it doesn’t hurt to do it multiple times (and in fact with a multiple-`pachd` cluster the scale-down will happen once for each `pachd`). Thoughts?

Reply:
With the spirit of making the logic easy to follow, I think that if it's not necessary for this feature to behave correctly, we shouldn't include it. 

To the second point, it isn't necessarily a "good thing" that all pachd's call this - perhaps if there were a lot of pachds, all the calls could overwhelm the Kube API Server. I don't know (and don't think) if that's a realistic concern.